_drafts/asking-for-help.md

```diff
@@ -0,0 +1,40 @@
+
+1. **Do some research before posting/asking**
+
+- The first step to encountering a problem is trying to solve it or at least understand it as much as you can on your own. Toss your error code on google, peruse help forums, try a few solutions. 
+
+- *Best case scenario:* Someone has already asked your exact question and the solution provided works for you! It's good practice to double-check that your question has not already been answered in an internet forum before posting.
+
+- *Worst case scenario:* No working results turn up. While this can be a frustrating process it will help you further diagnose your problem (if only by determining what doesn't work) and enable you to describe the situation more clearly. 
+
+2. **Clearly state your problem**
+
+- The quality of the answers you receive is directly related to the quality of the question posed. A good question is concise and systematically addresses the problem. Make sure that before posting any code that you properly introduce and give context to the question you are asking. Enough detail should be provided that someone could answer you without requesting more information.
+
+3. **Make sure your problem is reproducible**
+
+- It is best practice to provide a minimal reproducible example of the code that you need help with. An ideal example uses the minimal amount of code required to reproduce the problem and includes all the parts necessary to reproduce the problem including inputs and desired output. The test code should reliably reproduce the problem you're trying to solve. 
+
+- Do not use images of code. Copy and paste text and then format it as code. It helps with readability and makes it easier to run your test code.
+
+- Make sure that the code you show doesn't contain any sensitive information like personal information, passcodes, etc.
+
+4. **Be nice!**
+
+- Generally, the people you're going to for help are unpaid and volunteering their time and expertise. Kindness and understanding go a long way. 
+
+- Try and remember there are real humans behind the screen and brush off folks who may be less than welcoming.
+
+5. **Archive your discussion so it can be useful to others**
+
+- No one else should have to suffer through pages of purple links to find the answer they're looking for. It's always good practice to share back what you've learned. 
+
+- If you have requested help through a forum and a response solved your problem make sure you go back and mark the correct answer to let folks know what worked. It can even be helpful to note what answers did not work for you and why.
```

Comment:
Include some additional options here: file an issue for the content to be added to the wiki, or asking in slack means reporting back with the resolution